common/models/facility.js

```diff
@@ -518,11 +519,28 @@ module.exports = (Facility) => {
     http: { path: '/facilitieswithservice', verb: 'get' },
     accepts: [
       { arg: 'service_name', type: 'string' },
-      { arg: 'district_id', type: 'number' }
+      { arg: 'districts', type: 'array' }
     ],
     returns: [
       { arg: 'response', type: 'array' }
     ]
   });
 
+  Facility.totalFacilities = async (districts, cb) => {
+    const IDs = await getDistrictsIDs(districts);
+
+    const where =  {};
+    if (IDs.length) where.district_id = { inq: IDs }
+    const facilities = await Facility.find({where: where})
```

Comment:
consider a ternary operator in here: `await Facility.find({where: Ids.length ? { district_id: {inq: IDs} } : {}})` or, `const where = IDs.length ? { district_id: {inq: IDs } } : {}`.  In general, an empy (temp) variable like this can be easily refactored with a more declarative expression